crypto/crypto_aes_aesni.h

```diff
@@ -1,6 +1,8 @@
 #ifndef _CRYPTO_AES_AESNI_H_
 #define _CRYPTO_AES_AESNI_H_
 
+#include <emmintrin.h>
```

Comment:
Technically, `<emmintrin.h>` is all we need for `__m128i`, but since this is already inside an `_aesni.h` file, maybe it makes more sense to use `<wmmintrin.h>`?

And we also need `"cpusupport-config.h"` here.